template.yaml

```diff
@@ -7,11 +7,95 @@ Parameters:
     Type: String
     Default: "/SungardAS/Alerts/Account"
     Description: Account Cloudwatch Log Group Name
-  SubscriptionFilterDestinationArn:
+  DestinationArn:
     Type: String
     Description: Subscription Filter Destination Arn
+  DbUserName:
+    Type: String
+    Description: user name of mcaws db
+  DbPassword:
+    Type: String
+    NoEcho: true
+    Description: password of mcaws db
+  DbHost:
+    Type: String
+    Description: Host of mcaws db
+  MasterMgmAwsId:
```

Comment:
it should be MgmtAWSId.

Reply:
If I am going to change just name then i need to change a lot of places 
its required to again test with all scenario .its take more time

Reply:
ok do not change it.